app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+from app.errors import (
+    VaccineError,
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(
+            self,
+            visitor: dict
+    ) -> str | VaccineError | OutdatedVaccineError:
```

Comment:
You do not need to indicate exceptions in return type annotation, they are out of scope [for this feature](https://peps.python.org/pep-0484/#exceptions).